<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NX00605" failures="0" tests="8" name="Test" time="14.325" errors="0" timestamp="2023-10-12T18:49:17 PKT">
  <testcase classname="TestCases_Package.OrderAndCartPageTestCases" name="@BeforeTest initiateBrowser" time="3.527">
    <failure type="org.openqa.selenium.WebDriverException" message="java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:7750
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;NX00605&amp;apos;, ip: &amp;apos;10.44.0.45&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.7&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:7750
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'NX00605', ip: '10.44.0.45', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:92)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:277)
at Pages_Package.BaseClass.Setup_Browser(BaseClass.java:33)
at TestCases_Package.OrderAndCartPageTestCases.initiateBrowser(OrderAndCartPageTestCases.java:23)
Caused by: java.net.ConnectException: Failed to connect to localhost/[0:0:0:0:0:0:0:1]:7750
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:155)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
... 32 more
Caused by: java.net.ConnectException: Connection refused: no further information
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:542)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)
at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
at java.base/java.net.Socket.connect(Socket.java:633)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
... 52 more
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest initiateBrowser -->
  <testcase classname="TestCases_Package.ProdutctListingPageTestCases" name="@BeforeTest initiateBrowser" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeTest initiateBrowser -->
  <testcase classname="TestCases_Package.ReviewPageTestCases" name="@BeforeTest beforeMethod" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeTest beforeMethod -->
  <testcase classname="TestCases_Package.SignupPageTestCases" name="@BeforeTest initiateBrowser" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeTest initiateBrowser -->
  <testcase classname="TestCases_Package.LoginTestcases" name="@AfterTest Teardown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest Teardown -->
  <testcase classname="TestCases_Package.OrderAndCartPageTestCases" name="@AfterTest Teardown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest Teardown -->
  <testcase classname="TestCases_Package.ProdutctListingPageTestCases" name="@AfterTest TearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest TearDown -->
  <testcase classname="TestCases_Package.ReviewPageTestCases" name="@AfterTest Teardown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest Teardown -->
  <testcase classname="TestCases_Package.SignupPageTestCases" name="@AfterTest Teardown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest Teardown -->
  <testcase classname="TestCases_Package.LoginTestcases" name="Testcase_VerifyLoginwith_bothWrongEmailandPassword" time="0.0">
    <skipped/>
  </testcase> <!-- Testcase_VerifyLoginwith_bothWrongEmailandPassword -->
  <testcase classname="TestCases_Package.LoginTestcases" name="Testcase_VerifyLoginwith_bothCorrectEmailandPassword" time="0.0">
    <skipped/>
  </testcase> <!-- Testcase_VerifyLoginwith_bothCorrectEmailandPassword -->
  <testcase classname="TestCases_Package.LoginTestcases" name="usingSession_VerifyUserDoesnotExistInSystem_SignupNewForAccount" time="0.0">
    <skipped/>
  </testcase> <!-- usingSession_VerifyUserDoesnotExistInSystem_SignupNewForAccount -->
  <testcase classname="TestCases_Package.LoginTestcases" name="Verify_UserDoesnotExistInSystem_SignupNewForAccount" time="0.0">
    <skipped/>
  </testcase> <!-- Verify_UserDoesnotExistInSystem_SignupNewForAccount -->
  <testcase classname="TestCases_Package.ReviewPageTestCases" name="userIsAbleToSubmittReviweForDesktop" time="0.0">
    <skipped/>
  </testcase> <!-- userIsAbleToSubmittReviweForDesktop -->
  <testcase classname="TestCases_Package.ProdutctListingPageTestCases" name="f" time="0.0">
    <skipped/>
  </testcase> <!-- f -->
  <testcase classname="TestCases_Package.SignupPageTestCases" name="Testcases_ToRegisterUserSuccessfully" time="0.0">
    <skipped/>
  </testcase> <!-- Testcases_ToRegisterUserSuccessfully -->
  <testcase classname="TestCases_Package.OrderAndCartPageTestCases" name="VerifyplaceOrderofComputer" time="0.0">
    <skipped/>
  </testcase> <!-- VerifyplaceOrderofComputer -->
</testsuite> <!-- Test -->
